app/views/hyrax/leases/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Manage Leases</h1>
```

Comment:
Whatever is done for embargo can be done for leases - there _might_ be nearly identical partials.